freqtrade/optimize/backtesting.py

```diff
@@ -244,6 +244,21 @@ def load_bt_data(self) -> Tuple[Dict[str, DataFrame], TimeRange]:
         self.progress.set_new_value(1)
         return data, self.timerange
 
+    def get_required_startup(self, timeframe: str) -> int:
+        if not self.config['freqai']['enabled']:
```

Comment:
This must be `self.config.get('freqai', {}).get('enabled')`.
otherwise it's crashing with keyerror when you run with a "non-freqAI" enabled configuration.

Reply:
Ah I hit this during pytesting :-)